Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ghcr.io/mathisloge/mapnik:docker
```

Comment:
Is there a Dockerfile for this base image?

Reply:
Looks like it's build from this branch https://github.com/mapnik/mapnik/pull/4267